scipy/optimize/cython_optimize/__init__.py

```diff
@@ -0,0 +1,138 @@
+"""
+Cython Optimize Zeros API
+=========================
+The following four zeros functions can be accessed directly from Cython:
+
+- ``bisect``
+- ``ridder``
+- ``brenth``
+- ``brentq``
+
+Import the module into Cython as follows::
+
+    from scipy.optimize cimport cython_optimize
+
+The Cython Optimize Zeros API could be used to "vectorize" a root finder given
```

Comment:
"vectorize" isn't really the right word - using a for-loop is basically the opposite. Can this sentence just be left out? If you find it important, then something like "efficiently solve multiple ..."